spec/webview-spec.js

```diff
@@ -1061,6 +1061,12 @@ describe('<webview> tag', function () {
   })
 
   describe('media-started-playing media-paused events', () => {
+    before(function () {
+      if (!document.createElement('audio').canPlayType('audio/wav')) {
```

Comment:
If electrons ffmpeg build ever breaks for some reason then this test will not capture the issue anymore.

I'd prefer if you used the --grep and --invert flags to not run this test in your internal CI

Reply:
makes sense, we'll work around it in our CI then

Reply:
@MarshallOfSound hmm, actually wouldn't it be better to have a separate test, which checks if ffmpeg / list of supported formats works as expected? this test is whether a particular webContents event works, which cannot be verified if the format used to test it is not supported

Reply:
Having a separate test for that specifically probably makes sense.  You'll still have the same issue of having to disable the test yourself in your ci